Bottom: 3c68ae697f7c4e15e7346c72d61767c3b43fb14c
Top:    1b08efc20151978a5f24c054e8ab41df83a56e3d
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-06 09:09:16 -0500

Refresh of ansible-config

---
diff --git a/ansible-playbook/group_vars/docker-hosts/repos.yml b/ansible-playbook/group_vars/docker-hosts/repos.yml
deleted file mode 100644
index cc4d9cbf..00000000
--- a/ansible-playbook/group_vars/docker-hosts/repos.yml
+++ /dev/null
@@ -1,13 +0,0 @@
----
-
-buildbot_image_repo: https://github.com/AndrewCooper/docker-buildbot-master.git
-buildbot_worker_ntel_image_repo: https://github.com/AndrewCooper/docker-buildbot-worker-ntel.git
-buildbot_worker_ptxdist_image_repo: https://github.com/AndrewCooper/docker-buildbot-worker-ptxdist.git
-exim4_image_repo: https://github.com/AndrewCooper/docker-exim4.git
-gitlab_image_repo: 'https://github.com/AndrewCooper/docker-gitlab.git'
-mantisbt_image_repo: https://github.com/AndrewCooper/docker-mantisbt.git
-openldap_image_repo: https://github.com/AndrewCooper/docker-openldap.git
-openssl_image_repo: https://github.com/AndrewCooper/docker-openssl.git
-phpldapadmin_image_repo: https://github.com/AndrewCooper/docker-phpldapadmin.git
-svn_image_repo: https://github.com/AndrewCooper/docker-svn.git
-wiki_image_repo: https://github.com/AndrewCooper/docker-mediawiki.git
diff --git a/provision.sh b/provision.sh
deleted file mode 100755
index 3762e89d..00000000
--- a/provision.sh
+++ /dev/null
@@ -1,14 +0,0 @@
-#! /bin/bash
-
-#PYTHONUNBUFFERED=1 \
-     
-ANSIBLE_ENABLE_TASK_DEBUGGER=True \
-ANSIBLE_FORCE_COLOR=true \
-ANSIBLE_HOST_KEY_CHECKING=false \
-ANSIBLE_SSH_ARGS='-o UserKnownHostsFile=/dev/null -o IdentitiesOnly=yes -o ControlMaster=auto -o ControlPersist=60s' \
-exec ansible-playbook \
-        --connection=ssh \
-        --timeout=30 \
-        --inventory-file=ansible-playbook/staging \
-        -vv \
-        ansible-playbook/site.yml "$@"
